src/js/auto-advance.js

```diff
@@ -54,7 +63,19 @@ const setup = (player, delay) => {
   player.one('ended', player.playlist.autoadvance_.trigger);
 };
 
+/**
+ * Used to change the reset function in this module at runtime
+ * This should only be used in tests.
+ *
+ * @param {Function} fn
+ *        The function to se the reset to
+ */
+const setReset_ = (fn) => {
```

Comment:
This was added because the tests previously overwrote the reset function after import. This is technically illegal as anything that is imported is done so as a const, which would cause rollup to fail. This is a simple workaround for the tests.